tests/config/nf-test.config

```diff
@@ -31,7 +31,7 @@ profiles {
     docker {
         docker.enabled = true
         docker.fixOwnership = true
-        docker.runOptions = '-u $(id -u):$(id -g) --platform=linux/amd64'
```

Comment:
sure about that?

Reply:
It's a test for @mashehu 
https://nfcore.slack.com/archives/C050DNB6K5H/p1708330668655529

Reply:
Removing the `-u $(id -u):$(id -g)` seems to have fixed the remaining docker issues on the self hosted runners for the most part